rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -951,17 +953,24 @@ public RestPage<TaskStateData> getJobTaskStatesByTagPaginated(String sessionId,
 
     @Override
     public RestPage<TaskStateData> getJobTaskStatesByTagByStatusPaginated(String sessionId, String jobId, int offset,
-            int limit, String taskTag, String statusFilter) throws RestException {
+            int limit, String taskTag, PathSegment statusFilter) throws RestException {
         if (limit == -1)
             limit = TASKS_PAGE_SIZE;
         try {
             Scheduler s = checkAccess(sessionId, PATH_JOBS + jobId + "/taskstates/" + taskTag + "/paginated");
             JobState jobState = s.getJobState(jobId);
-            TaskStatesPage page = jobState.getTaskByTagByStatusPaginated(offset, limit, taskTag, statusFilter);
+            TaskStatesPage page = jobState.getTaskByTagByStatusPaginated(offset,
+                                                                         limit,
+                                                                         taskTag,
+                                                                         URLDecoder.decode(statusFilter.getPath(),
+                                                                                           StandardCharsets.UTF_8.name()));
```

Comment:
I would prefer if you use:
Charset.forName(PASchedulerProperties.FILE_ENCODING.getValueAsString())